lib/pure/asyncfile.nim

```diff
@@ -123,7 +123,7 @@ proc openAsync*(filename: string, mode = fmRead): AsyncFile =
 
     result = newAsyncFile(fd.AsyncFd)
 
-proc readBuffer*(f: AsyncFile, buf: pointer, size: int): Future[int] =
+proc readBuffer*(f: AsyncFile; buf: pointer, size: int): Future[int] =
```

Comment:
This is wrong, there are commas used above. Also, only one comma was changed to a semicolon?

In any case, I am a comma lover and strongly believe Nimpretty should default to commas. I don't get this push to use semicolons everywhere, as far as I understand it fixes some edge case where the compiler needs some help disambiguating, so why use it everywhere?

Reply:
This also extends to all the changes in asyncdispatch above. Please revert these.

Reply:
> This is wrong, there are commas used above. Also, only one comma was changed to a semicolon?

I finally managed to find the reason for this behaviour (both changing comma to semicolon when it shouldn't, and then afterwards not changing it once it has started changing it).

Bugfix + more conservative behaviour incoming.